frontend/src/Components/dashboard/reciever/Home.jsx

```diff
@@ -392,7 +394,16 @@ const Home = () => {
                 initialViewState={mapState}
                 style={{ width: 600, height: 340 }}
                 mapStyle="mapbox://styles/mapbox/streets-v9"
+
               >
+                  <Marker 
+                    longitude={longitude}
+                    latitude={latitude}
+                    anchor="bottom"
+                  >
+                  <img src={pin} alt="pin" />

```

Comment:
Could you please add `style={{ width: "26px", height: "40px" }}` to the `img` tag? The default style making it a bit larger.